google-cloud-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

```diff
@@ -245,6 +250,7 @@ public void close() throws Exception {
     configApi.close();
     loggingApi.close();
     metricsApi.close();
+    providerManager.getChannel().shutdown();
```

Comment:
You are right. Although I would keep the `close()` calls in case we end up adding other Api-scoped closeables.